components/OneDay/Partners.js

```diff
@@ -7,9 +7,13 @@ import TopPartners from '../shared/TopPartners';
 const PARTNER_LEVELS_TO_DISPLAY = ['CORPORATE_PARTNER', 'PARTNER'];
 
 const Partners = ({ partners }) => {
-  const topPartners = partners.filter(partner =>
-    PARTNER_LEVELS_TO_DISPLAY.includes(partner.level),
-  );
+  const topPartners = partners
```

Comment:
To sort these you should use lodash `sortBy`. We use it throughout the app. Will allow you to sort by level and then say alpha if wanted.

Reply:
Last commit uses lodash for sort. Please review @saragibby